guide/onboarding/protecting-a-wallet.md

```diff
@@ -0,0 +1,223 @@
+
+However, users with poor security practices might create easy-to-guess PINs (such as 1234), which would decrease the level of security. This design also relies on users to remember their PIN correctly.
+
+{% include tip/open.html color="green" icon="check" label="Do: Educate on Good Security Practices" %}
+
+Within your product upon PIN creation, tell your users what a good, secure PIN is (i.e. not their birthday). Recommend against (or even make it impossible) to create a PIN such as 1234, 1111, etc.
+
+{% include tip/close.html %}
+
+
+### Hiding Sensitive Information
+
+Imagine this scenario. You are in a public place, and you need to make a payment using your bitcoin wallet. You open your wallet on your phone, but you don’t feel comfortable having your address and balance information clearly visible to strangers who may be looking over your shoulder, persons lurking or video surveillance . Hence by giving users the ability to hide sensitive information in their wallet if desired, they gain an added sense of physical privacy and security [^3]when using the app in public.
+
+### What information is considered sensitive ?
+
+Sensitive information in wallet applications include the wallet balance, addresses, private keys and previous transactions information.
+- Wallet Balance - shows how much you own
+- Addresses - can be used to track your transaction history
+- Private keys - can be used to access and transfer your bitcoins
+
+It's more common for wallets to protect private keys but not much is done for other sensitive information like the balance, addresses and previous transactions. A few wallets like Bitcoin Core, Wasabi, Wallet of Satoshi, and others have made it work though. Below describes a pattern, and considerations for hiding and revealing sensitive information.
+
+### Quickly hide from overview screen
+
+The hide icon / button, which is usually displayed within close reach of the balance itself, is used to quickly and easily hide and reveal wallet information by tapping or clicking on it.[^2]
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   retina = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   mobile = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   mobileRetina = "/assets/images/guide/onboarding/protecting-a-wallet/hide-by-eye-icon.gif"
+   alt-text = "hide-by-eye icon"
+   width = 1600
+   height = 800
+%}
+
+#### Pros
+
+- It is quite convenient to switch between revealed and hidden state
+
+#### Cons
+
+- Easy for anyone else to reveal your information if they have access to your device.
+
+
+## Entering a PIN to reveal information
```

Comment:
Does this slightly feel like a duplication of the "Pin" section?

Reply:
Yeah could argue that. I think it's okay though as this provides a more specific case for revealing a users balance. Initially is more for just locking the whole app.

Reply:
Agreed, not really a blocker for the current phase for sure.